tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -26,14 +29,25 @@ def tap_save(self):
         Wait(self.marionette).until(lambda m: save_button.get_attribute('disabled') == 'false')
         save_button.tap()
 
+    def tap_exit(self):
+        self.marionette.find_element(*self._header_locator).tap(25, 25)
+
+    def cancel_share(self):
+        self.marionette.find_element(*self._actions_cancel_locator).tap()
+
     @property
     def ring_tones(self):
         return [self.RingToneItem(self.marionette, item)
                 for item in self.marionette.find_elements(*self._ring_tone_locator)]
 
+    @property
+    def screen_element(self):
```

Comment:
Why is this called screen_element, when the locator is called _page_locator (which is actually referencing an element with id=list-parent)?